Documentation/markdown/IFC4.3XListOfChanges.md

```diff
@@ -0,0 +1,33 @@
+# List of changes between IFC4X3_RC4 and IFC4X3 versions
+
+### Missing data in OIP
+
+| Name                                 | IFC4X3_RC4.exp          | IFC4X3.exp            |
+|--------------------------------------|-------------------------|----------------------|
+| IfcCosine                            | `:heavy_check_mark:`    | `:x:`                |
+| IfcSine                              | `:heavy_check_mark:`    | `:x:`                |
+| IfcCosineSpiral                      |  `:x:`                  | `:heavy_check_mark:` |
+| IfcSineSpiral                        |  `:x:`                  | `:heavy_check_mark:` |
+
```

Comment:
please check if ifcGradient, ifcPointDim and ifcSegmentDim are relevant for us
There are also some entities related to textures. Currently we don't support textures but technically it would be relevant for OIP. Maybe you can include them. (IfcIndexedPolygonalTextureMap, IfcTextureCoordinateIndices, IfcTextureCoordinateIndicesWithVoids) Take a closer look at them. I just guessed by their name if they could be relevant.